### Calculadora de interes compuesto Nu

- ¿Como usarla?

Completa tus datos: 


Primero validaremos cuanto dinero deposita diariamente Nu:

- Realizamos un experimento depositando $6,600 pesos en una cuenta de ahorro en junio de 2024 y obteniendo 2.66 como ingreso por un dia

Por ende haciendo la division de  $ 2.4 / 6600 $ nos da como resultado la variable * interes_diario_por_peso *

In [50]:
print(2.66 / 6600)

0.0004030303030303031


In [51]:
interes_diario_por_peso = 0.0004030303030303031
Inversion = 10000

Comprobaremos el interes que ofrece Nu vs lo real

In [52]:
InteresAnualNuOfertadoJunio2024 = 14.5

In [53]:
interes_real_nu = 360 * interes_diario_por_peso
interes_real_nu

0.1450909090909091

Cumple con lo prometido Nu de interes

### Primera prueba: 

Si al mes ingreso 10,000 pesos , y no retiro mi ganancia más lo ya previamente invertido. ¿Cuanto tendre a final de el periodo?


Primero haremos el calculo por un mes simple

In [54]:
Ingreso_por_dia_al_mes = Inversion * (1 + interes_diario_por_peso)**30
Ingreso_por_dia_al_mes

10121.61834145746

Nos da como resultado $121.618 pesos de ganancia por interes

Ahora por los *6 meses* del ejercicio

In [55]:
for i in range(0, 360):
    #El primer mes se empieza con 10,000 y se termina con 10,000 + 30 días de intereses
    final_mes = Inversion * (1 + interes_diario_por_peso)**30
    Inversion = final_mes + 10000
    
    print(f"Mes {i}: {final_mes}")
    print(f"Interes: {final_mes - (10000*(i+1))}")

Mes 0: 10121.61834145746
Interes: 121.6183414574607
Mes 1: 20366.334126470272
Interes: 366.33412647027217
Mes 2: 30735.644445730715
Interes: 735.6444457307152
Mes 3: 41231.06459729977
Interes: 1231.0645972997736
Mes 4: 51854.12830804214
Interes: 1854.1283080421417
Mes 5: 62606.387957754254
Interes: 2606.387957754254
Mes 6: 73489.41480601816
Interes: 3489.4148060181615
Mes 7: 84504.79922181435
Interes: 4504.799221814348
Mes 8: 95654.1509159271
Interes: 5654.150915927094
Mes 9: 106939.09917617623
Interes: 6939.099176176227
Mes 10: 118361.29310550985
Interes: 8361.293105509845
Mes 11: 129922.40186299256
Interes: 9922.401862992556
Mes 12: 141624.1149077247
Interes: 11624.114907724696
Mes 13: 153468.142245728
Interes: 13468.142245727999
Mes 14: 165456.21467983376
Interes: 15456.214679833764
Mes 15: 177590.08406261032
Interes: 17590.084062610316
Mes 16: 189871.52355236636
Interes: 19871.523552366358
Mes 17: 202302.32787226784
Interes: 22302.327872267837
Mes 18: 214884.31357260616
Interes: 24

Al finalizar el mes 6 obtendremos $2606.387 pesos de ganancia

In [56]:
202302.32787226784 - 180000

22302.327872267837

Siendo regulares por 18 meses habremos ganado $22,302 pesos de solo interes

In [57]:
550522.8489022581 - 360000

190522.8489022581

Siendo regulares por 36 meses habremos obtenido 766,387.85 pesos, lo que significa que habremos ganado 406,387.85 pesos en intereses.